draft-ZIP-0226.rst

```diff
@@ -0,0 +1,303 @@
+::
+
+  ZIP: 226
+  Title: Transfer and Burn of Zcash Shielded Assets
+  Owners: Daniel Benarroch <daniel@qed-it.com>
+          Pablo Kogan <pablo@qed-it.com>
+          Aurelien Nicolas <aurel@qed-it.com>
+  Credits: Daira Hopwood
+           Jack Grigg
+           Shahaf Nacshon
+           Vivek Arte
+  Status: Draft
+  Category: Consensus
+  Created: 2022-05-01
+  License: MIT
+  Discussions-To: <https://github.com/zcash/zips/issues/618>
+
+
+Terminology
+===========
+
+The key word "MUST" in this document is to be interpreted as described in RFC 2119 [#RFC2119]_.
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [#zip-0200]_.
+
+The term "Orchard" in this document is to be interpreted as described in ZIP 224 [#zip-0224]_.
+
+We define the following additional terms:
+
+- Asset: A type of note that can be transferred on the Zcash block chain. ZEC is the default (and currently the only defined) asset for the Zcash mainnet.
+- Zcash Shielded Asset: an asset with issuance defined on the Zcash block chain, and, for now, belonging to the Orchard anonymity pool.
+- Wrapped Asset: a ZSA asset with native issuance defined outside the Zcash block chain.
+
+Abstract
+========
+
+ZIP 226 and ZIP 227 propose in conjunction the Zcash Shielded Assets (ZSA) protocol, which is an extension of the
+Orchard protocol that enables the creation, transfer and burn of custom assets on the Zcash chain. The creation of such assets is defined
+in ZIP 227 [#zip-0227]_. The transfer and burn of such assets is defined in ZIP 226 [#zip-0226]_.
+
+Motivation
+==========
+
+The current Orchard protocol does not support custom assets, which is an important aspect for the proper evolution of the Zcash ecosystem. Enabling multi-asset support on the Zcash chain will open the door for a host of applications and enhance the ecosystem with application developers and asset custody institutions for issuance and bridging purposes.
+
+Overview
+========
+In order to be able to represent different asset types in Orchard, one needs to define a new data field that uniquely represents the *type* of the asset in question, which we call :math:`\mathsf{type}`.
+
+The type will be used to enforce the balance of an action description [#protocol-actions]_ is preserved across assets (see the Orchard Binding Signature [#protocol-binding]_ , and by extension of an Orchard transaction. Mainly, the sum of all the :math:`\mathsf{value^{net}}`, as :math:`\mathsf{value^{old}-value^{new}}`, derived from each action description, must be balanced only **with respect to the same asset type**. This is specially important since we will allow different action descriptions to transfer notes of different asset types, where the overall balance is checked without revealing which assets (or how many different types) are being transferred.
```

Comment:
```suggestion
The type will be used to enforce the balance of an Action Description [#protocol-actions]_ is preserved across Assets (see the Orchard Binding Signature [#protocol-binding]_), and by extension the balance of an Orchard transaction. That is, the sum of all the :math:`\mathsf{value^{net}}` from each Action Description, computed as :math:`\mathsf{value^{old}-value^{new}}`, must be balanced only **with respect to the same Asset type**. This is specially important since we will allow different Action Descriptions to transfer notes of different Asset types, where the overall balance is checked without revealing which Assets (or how many different types) are being transferred.
```